test/jdk/java/awt/Window/FullWindowContentTest/FullWindowContentRenderTest.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021, JetBrains s.r.o.. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
+ * @test
+ * @key headful
+ * @bug 8258788
+ * @summary [macosx] full window content option rendering test
```

Comment:
The test passed even before the fix.

Reply:
It's strange. Just rerun it with the following command line:

jtreg -Dsun.java2d.metal=True -testjdk:/Users/user/ws/jbr-dev/build/macosx-x86_64-server-release/images/jdk-bundle/jdk-17.jdk/Contents/Home test/jdk/java/awt/Window/FullWindowContentTest/FullWindowContentRenderTest.java

Test results: failed: 1

Reply:
What OS version do you have?

Reply:
macOS 11.2(20D64), did you set the sRGB color profile in the settings?

Reply:
Yes, it's IEC61966-2.1

Reply:
Ok, found the issue on my side, by default the test uses the OGL, not a metal.

Reply:
Good news! @mrserb Could you sponsor this pull request?